pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -271,20 +278,28 @@ type ACMEIssuerDNS01ProviderAkamai struct {
 type ACMEIssuerDNS01ProviderCloudDNS struct {
 	// +optional
 	ServiceAccount *cmmeta.SecretKeySelector `json:"serviceAccountSecretRef,omitempty"`
-	Project        string                    `json:"project"`
+
+	// +kubebuilder:validation:Required
```

Comment:
This change doesn't seem to do anything - fields are already considered `required` if they are not `optional`